src/game/UICmdList.h

```diff
@@ -0,0 +1,60 @@
+// UICmdListDlg.h: interface for the CUICmdListDlg class.
+//
+//////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include "N3Base/N3UIBase.h"
+#include "N3Base/N3UIButton.h"
+#include "N3Base/N3UIList.h"
+#include "N3Base/N3UIEdit.h"
+
+//////////////////////////////////////////////////////////////////////
+
+class CUICmdListDlg : public CN3UIBase {
+    bool               m_bDestoyDlgAlive;
+    class CUICmdEdit * m_pUICmdEdit;
+    CN3UIButton *      m_pBtn_cancel;
```

Comment:
Based on the naming scheme of the others, this should probably be `m_pBtn_Cancel`